main.tf

```diff
@@ -19,7 +19,7 @@ resource "random_id" "random_hash_suffix" {
 }
 
 resource "null_resource" "org_id_and_folder_id_are_both_empty" {
-  count = "${var.org_id == "" && var.folder_id == "" ? 1 : 0}"
+  count = "${length(var.composite_root_resources) == 0 && var.org_id == "" && var.folder_id == "" ? 1 : 0}"
```

Comment:
@umairidris @morgante We're working on the integration tests for the upcoming 0.12 release and running into a number of the following errors.  Digging into them, we're unsure why this null resource fails when composite_root_resources has items in the list.  Could you help me understand the intent of adding this variable to the list?

```
Error: Error running command 'false': exit status 1. Output:
```

Reply:
No rush on this, we traced the Error to another null_resource, `module.server.null_resource.missing_emails`

Reply:
Please open a separate issue instead of commenting on merged PRs.